action.yml

```diff
@@ -216,6 +216,7 @@ runs:
         tags: |
           # SHA for all events (no nightly for GAR)
           type=raw,value=${{ steps.setup.outputs.commit_sha }},prefix=${{ inputs.tag_prefix }},suffix=${{ inputs.tag_suffix }},priority=2000
+          type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
I believe `latest` is only supposed to be applied when we perform a released version?
`nightly` would be the one intended to always point to the default branch.

Reply:
We actually use latest for images in artifact registry in getsentry and relay when I checked:
Getsentry: https://github.com/getsentry/getsentry/blob/master/cloudbuild.yaml
Relay: https://github.com/getsentry/relay/blob/ba17d8ad8716e3071e7f52094dc1a82639ef23e4/.github/workflows/ci.yml#L560

Reply:
Seems reasonable to keep this consistent with how things are done today to avoid breakages